tests/unit/components/DocumentationTopic/PrimaryContent/DeclarationToken.spec.js

```diff
@@ -97,4 +98,31 @@ describe('DeclarationToken', () => {
     expect(token.props('tokens')).toEqual(propsData.tokens);
     expect(token.props('kind')).toEqual(propsData.kind);
   });
+
+  it('renders a `SyntaxToken` for basic `attribute` tokens', () => {
+    const propsData = {
+      kind: TokenKind.attribute,
+      text: '@foo',
+    };
+    const wrapper = mountToken({ propsData });
+    const token = wrapper.find(SyntaxToken);
+    expect(token.exists()).toBe(true);
+    expect(token.props('kind')).toBe(TokenKind.attribute);
+    expect(token.props('text')).toBe(propsData.text);
+  });
+
+  it('renders a `LinkableToken` for `attribute` tokens with an `identifier`', () => {
+    const propsData = {
+      kind: TokenKind.attribute,
+      identifier: '@foo',
+      text: '@foo',
```

Comment:
Can we use a different `identifier` vs `text` for this test? Just to make sure it's not accidentally getting things right.

Reply:
e568d4e8887ab3ce070df2861b1b8722b3f7509d